packages/bpk-component-mobile-scroll-container/src/BpkMobileScrollContainer.js

```diff
@@ -97,17 +97,19 @@ type State = {
 };
 
 const propTypes = {
+  ariaLabel: PropTypes.string,
   children: PropTypes.node.isRequired,
   scrollerRef: PropTypes.func,
   innerContainerTagName: PropTypes.string,
   className: PropTypes.string,
   leadingIndicatorClassName: PropTypes.string,
   trailingIndicatorClassName: PropTypes.string,
-  style: PropTypes.object,  
+  style: PropTypes.object,
   showScrollbar: PropTypes.bool,
 };
 
 const defaultProps = {
+  ariaLabel: '',
```

Comment:
I don't think this is the correct approach because it modifies behaviour of consumers that don't use the prop unnecessarily.
The value should be able to be null if not supplied, in which case it should not apply the arialabel tag to the element.
This way it would provide the flexibility for consumers to give "" as the string if they want to but also maintain the old behaviour of not adding it to consumers that don't use it.